### ISSUE TITLE
Showing multiple sUtras operating simultaneously

### ISSUE THREAD

**Issue by drdhaval2785** [drdhaval2785/SanskritVerb#148]
Right now we are handling only one sUtra / vArtika in storedata function and display_from_storedata function.
There are some genuine cases where there is simultaneous application of two sUtras to give one output. e.g. uraNraparaH and iko yaNaci.
In such cases we would have to find out some way to display two sUtras simultaneously.

**Comment by drdhaval2785** [comment]
The same issue arose in juhotyAdi derivation in #200 .

**Comment by drdhaval2785** [comment]
The only solution I see is to store the sUtra numbers in array rather than string form in storedata function and show all the members of the array by comma separation.
Still need to work out little details.

**Comment by akprasad** [comment]
This is possible with vidyut-prakriya. Before supporting it, I would like to know of more examples where showing multiple rules would be good user experience.

**Comment by drdhaval2785** [comment]
It is not about user experience. It is because of the fact that without effect of the two sUtras, the derivation will be faulty. Both have to apply simultaneously.

**Comment by akprasad** [comment]
Sorry, let me rephrase. What I mean is that to me, it seems that there are many such sutras one could include in the name of simultaneous application, e.g. ङिच्च, आद्यन्तौ टकितौ. etc. Do you wish to include these as well?

Likewise, what is your view on including अधिकार-सूत्रs like आद्युदात्तश्च?
